datasync/downloader/downloader_test.go

```diff
@@ -75,6 +80,8 @@ type downloadTester struct {
 func newTester() *downloadTester {
 	testdb := database.NewMemoryDBManager()
 	genesis := blockchain.GenesisBlockForTesting(testdb, testAddress, big.NewInt(1000000000))
+	params.SetStakingUpdateInterval(4)
+	reward.SetTestStakingManagerWithDB(testDB)
```

Comment:
Can we rollback both StakingUpdateInterval and StakingManager to original values after testing?  e.g. in `tester.terminate()`

sample: https://github.com/klaytn/klaytn/blob/v1.8.4/consensus/istanbul/backend/engine_test.go#L755

Reply:
Hmm.. I have updated for setting and rollback governance for testing purpose. Since staking manager is global variable, I have updated differently. Please take another look.

Reply:
@jeongkyun-oh Nice job handling concurrent tests!